projects/plugins/jetpack/extensions/blocks/ai-assistant/upgrade-prompt.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import React from 'react';
+/*
+ * Internal dependencies
+ */
+import { Nudge } from '../../shared/components/upgrade-nudge';
+
+const UpgradePrompt = () => {
+	return (
+		<Nudge
+			buttonText={ 'Upgrade' }
+			checkoutUrl={ 'admin.php?page=my-jetpack#/add-jetpack-ai' }
```

Comment:
You can use the `Jetpack_Editor_Initial_State` global object here:

```suggestion
			checkoutUrl={ `${ Jetpack_Editor_Initial_State.adminUrl } ?page=my-jetpack#/add-jetpack-ai` }
```

Something like this ^

Reply:
Addressed in 3481fba980f7cab15851c51cf56e401c587bc1d9